### DISCUSSION TITLE
Separate config, data and log directories

### DISCUSSION THREAD

**Discussion by lgLindstrom** [Koenkk/zigbee2mqtt#7773] [General]
Hi

I am running Zigbee2mqtt in a Docker container on a RasperryPi host. 
Is there a way to separate the config, data and logger directory? I would like to map this to different disks on my docker host.

If not, please see this as a feature request.

**Comment by Koenkk** [comment]
See the `log_directory` option: https://www.zigbee2mqtt.io/information/configuration.html
